App error: [2026-05-24T02:20:29.617Z] [BATCH] [ERROR] Weekly relation validation failed {

<!-- memento-log-monitor:fingerprint=f6d543e02eafcae2 -->
## 운영 감지 요약

- Occurrences: 1
- First seen: 2026-05-24T02:24:47.623Z
- Last seen: 2026-05-24T02:24:47.623Z
- Severity: error
- Source: app-log
- Fingerprint: f6d543e02eafcae2

## 최근 로그

### 2026-05-24T02:24:47.623Z

```text
[2026-05-24T02:20:29.617Z] [BATCH] [ERROR] Weekly relation validation failed {
```

원본 전체 로그는 로컬 `log-issue-monitor` 상태 디렉터리에만 보존됩니다.
<!-- /memento-log-monitor -->

--- a/scripts/generate-relation-report.ts
+++ b/scripts/generate-relation-report.ts
@@ -13,12 +13,15 @@
 import { readFileSync, writeFileSync } from 'fs';
 import { join } from 'path';
 import Database from 'better-sqlite3';
-import { RelationExtractor } from '../src/services/relation-extractor.js';
-import { RelationQualityValidator } from '../src/services/relation-quality-validator.js';
-import type { ExpectedRelation, ExtractedRelation } from '../src/services/relation-quality-validator.js';
-import { DatabaseUtils } from '../src/utils/database.js';
-import { RelationEngineSchemaMigration } from '../src/database/migration/migrations/005-relation-engine-schema.js';
-import type { MemoryItem } from '../src/types/index.js';
+import { RelationExtractor } from '../packages/memento-core/src/domains/relation/services/relation-extractor.js';
+import {
+  RelationQualityValidator,
+  type ExpectedRelation,
+  type ExtractedRelation,
+} from '../packages/memento-core/src/domains/relation/services/relation-quality-validator.js';
+import { DatabaseUtils } from '../packages/memento-core/src/shared/utils/database.js';
+import { RelationEngineSchemaMigration } from '../packages/memento-core/src/infrastructure/database/database/migration/migrations/005-relation-engine-schema.js';
+import type { MemoryItem } from '../packages/memento-core/src/shared/types/index.js';
 
 /**
  * 명령줄 인자 파싱

--- a/scripts/weekly-relation-validation.ts
+++ b/scripts/weekly-relation-validation.ts
@@ -12,12 +12,15 @@
 import { readFileSync, writeFileSync, mkdirSync, existsSync } from 'fs';
 import { join } from 'path';
 import Database from 'better-sqlite3';
-import { RelationExtractor } from '../src/services/relation-extractor.js';
-import { RelationQualityValidator } from '../src/services/relation-quality-validator.js';
-import type { ExpectedRelation, ExtractedRelation } from '../src/services/relation-quality-validator.js';
-import { DatabaseUtils } from '../src/utils/database.js';
-import { RelationEngineSchemaMigration } from '../src/database/migration/migrations/005-relation-engine-schema.js';
-import type { MemoryItem } from '../src/types/index.js';
+import { RelationExtractor } from '../packages/memento-core/src/domains/relation/services/relation-extractor.js';
+import {
+  RelationQualityValidator,
+  type ExpectedRelation,
+  type ExtractedRelation,
+} from '../packages/memento-core/src/domains/relation/services/relation-quality-validator.js';
+import { DatabaseUtils } from '../packages/memento-core/src/shared/utils/database.js';
+import { RelationEngineSchemaMigration } from '../packages/memento-core/src/infrastructure/database/database/migration/migrations/005-relation-engine-schema.js';
+import type { MemoryItem } from '../packages/memento-core/src/shared/types/index.js';
 
 /**
  * 명령줄 인자 파싱

